packages/bpffs/bpffs.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Mount BPF filesystem
+DefaultDependencies=no
+Before=local-fs.target umount.target
+After=swap.target
```

Comment:
I appreciate the attention to detail in setting this up as a separate package. That would be valuable if for some reason it's not safe for all variants of Bottlerocket.

I need to spend some time looking at the implications of bpffs and whether additional controls (e.g. via SELinux policy) are needed, but my initial take is that we'll want this file to go into `release` directly.

Reply:
Ok great! Let me know what you find out and what you'd like from me and I'll make it happen! Thanks so much for looking into this 😄

Reply:
Let's add this file as `packages/release/bpffs.mount` and update `release.spec` with:
```
Source1013: bpffs.mount
```

Then it can be installed along with other units like `Source1012` / `%{S:1012}`.

Reply:
On it now! 😄

Reply:
All set @bcressey! I haven't done any further testing after this change – do you have a process you follow? There are quite a few image variants and it would probably take considerable time for me to test them all… I can re-publish an AMI and re-run my EKS smoke test with Cilium if you'd like though! Let me know what else you need from me! 🎉

Reply:
We run more extensive tests on all variants prior to release, so you don't need to worry about testing all of them.

It'd be good to confirm at least that `/sys/fs/bpf` is mounted as you expect on the host.

You could also run through the [Cilium connectivity test](https://docs.cilium.io/en/v1.8/gettingstarted/k8s-install-eks/#deploy-the-connectivity-test) and confirm that it passes. Checking `dmesg` afterwards for errors, especially SELinux denials, also wouldn't hurt.

(Any problems from the more thorough testing wouldn't be blockers here, as long as we had issues to track them.)

Reply:
Ok sure I'll do that! It might take a little bit of time here yet but we'll make it happen!

Also what is your release process like? After we (hopefully 🤞 ) merge this should I expect about a week? month? until this functionality is usable in the released bottlerocket AMIs that are listed by `"/aws/service/bottlerocket/aws-k8s-${var.eks_cluster.version}/x86_64/latest/image_id"`?

Reply:
We don't have a fixed release cadence but roughly 4-6 weeks has been typical.